### PR TITLE
Example domain change; Requirements in separate doc

### PR DESCRIPTION
**Pull request by AbelLykens**
Changed example domain to example.org

Moved requirements to separate REQUIREMENTS document, I feel the instructions in BRIDGE were incorrectly placed, having a separate requirements document makes more sense to me.

### CODE REVIEW

**Comment by AbelLykens** [comment]
It appears I did not use add the index branch correctly, now it added to this PR.

**Comment by andrerfneves** [comment]
So we've decided to move the `bridgeAddr` and the new `Satdress` codebases and repos and documentation to their individual pages. This way this readme just stays as a lean provider of info and links to specific resources. Check out the new Bridge.md.

File: BRIDGE.md [hunk 1]
```diff
@@ -6,11 +6,11 @@
 
 Simple plug-n-play server for supporting [Lightning Address](https://lightningaddress.com) protocol.
 
-> At the end of this flow you will be able to receive payments at `any_name@domain.com`.
+> At the end of this flow you will be able to receive payments at `any_name@example.org`.
 
 ## General
 
-A server that allows you to receive payments at `yourname@yourdomain.com` noncustodially. The Bridge Server serves the necessary JSON and then uses RPC calls to connect to your node and fetch invoices on demand.
+A server that allows you to receive payments at `yourname@yourexample.org` noncustodially. The Bridge Server serves the necessary JSON and then uses RPC calls to connect to your node and fetch invoices on demand.
```
Comment (Fittiboy) [inline BRIDGE.md:13]: I think in this specific case, `domain` makes more sense than `example`, but I also prefer `.org`

Comment (andrerfneves) [reply BRIDGE.md:13]: Technically `.org` `.com`, or any TLD would work here. `.com` is just the most widely known so no way to misunderstand what it represents. Probably best to stick to the most known for this.

**Pull request closed without merging**